libbpf-tools/slabratetop.bpf.c

```diff
@@ -0,0 +1,54 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+/* Copyright (c) 2022 Rong Tao */
+#include <vmlinux.h>
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+#include <bpf/bpf_tracing.h>
+#include "slabratetop.h"
+#include "stat.h"
```

Comment:
You don't need this.

Reply:
> You don't need this.

Yep, remove it in https://github.com/iovisor/bcc/pull/3797/commits/f6ac2c9fa41214d54a749ecb42973d63af9b7d3a